### PR TITLE
85 add dynamic import to open positions freshteam widget

### PR DESCRIPTION
**Pull request by TowerOneka**
#85

### CODE REVIEW

File: site/src/pages/_app.tsx [hunk 1]
```diff
@@ -18,6 +18,12 @@ import { AppProps } from 'next/app'
 type ComponentWithLayout = AppProps['Component'] & {
   getLayout?: (page: JSX.Element) => JSX.Element
 }
+;(async () => {
```
Comment (ukorvl) [inline site/src/pages/_app.tsx:21]: What if we just import this styles as usual, but on a particular `careers` page?

Comment (TowerOneka) [reply site/src/pages/_app.tsx:21]: https://nextjs.org/docs/messages/css-global

Comment (KlonD90) [reply site/src/pages/_app.tsx:21]: I am pretty sure if i will visit a page from main page i've got a not a style. You should check url in another way.

Comment (TowerOneka) [reply site/src/pages/_app.tsx]: It will work, because page was reloaded on careers page, but I did refactoring to solve it another way

**Review by ukorvl — APPROVED** [review]

**Pull request merged**